<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIUser</class>
 <widget class="QWidget" name="GUIUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIUser</string>
  </property>
  <widget class="QListWidget" name="watchList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addWButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="presenterLabel">
      <property name="text">
       <string>Presenter</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="presenterEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="fileNameLabel">
      <property name="text">
       <string>File name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="fileNameEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="displayButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
  </widget>
  <widget class="QPushButton" name="watchButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Watch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tableButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
